tests/apps/search/test_templates_search.py

```diff
@@ -33,7 +33,6 @@ def test_templates_search_content(self):
                 (
                     r"<div[\\n ]*"
                     r'class="richie-react richie-react--search"[\\n ]*'
-                    r'data-locale="en_US"[\\n ]*'
```

Comment:
replace it by a test that ensures the locale is in the set in the <html> tag as expected?